src/Backoffice/CIties/App/Transformers/CityTransformer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Cities\App\Transformers;
+
+use Flugg\Responder\Transformers\Transformer;
+use Illuminate\Database\Eloquent\Collection;
+use Lightit\Backoffice\Cities\Domain\Models\City;
+use Lightit\Backoffice\Flights\Domain\Models\Flight;
+
+class CityTransformer extends Transformer
+{
+    /**
+     * @return array{id: int, name: string, arriving_flights: Collection<int, Flight>, departing_flights: Collection<int, Flight>}
+     */
+    public function transform(City $city): array
+    {
+        return [
+            'id' => $city->id,
+            'name' => $city->name,
+            'arriving_flights' => $city->arrivingFlights,
+            'departing_flights' => $city->departingFlights,
+        ];
+    }
```

Comment:
```suggestion
    protected $load = [
        'arriving_flights',
        'departing_flights'
    ];

    /**
     * @return array{id: int, name: string, arriving_flights: Collection<int, Flight>, departing_flights: Collection<int, Flight>}
     */
    public function transform(City $city): array
    {
        return [
            'id' => $city->id,
            'name' => $city->name,
        ];
    }
```
This is how to load relationships using the package.